src/api/app/models/user_ldap_strategy.rb

```diff
@@ -2,281 +2,334 @@
 class UserLdapStrategy
   @@ldap_search_con = nil
 
-  def is_in_group?(user, group)
-    user_in_group_ldap?(user.login, group)
-  end
-
-  def local_role_check(role, object)
-    local_role_check_with_ldap(role, object)
-  end
+  class << self
+    # This static method tries to find a group with the given gorup_title to check whether the group is in the LDAP server.
```

Comment:
I try to avoid it so you can review without whitespace...

Reply:
That is nice from you... but you could always add the typo fix in other commit. This way:
- the "moving methods around" commit is separated from the small typo fix commit
- refactoring changes are keep together in the same pull request.

I believe, that giving the small size of the typo fix commit it can be easily reviewed, and is not worth it to include it in other pull request. What do you think?

Reply:
> I think it doesn't really matter, you can do it either way. Its a nitpick.

Let's agree then, that we are going to do it. :+1:

> What does matter is that someone reviews the content of this PR....

You already got my review.

Reply:
> You already got my review.

I don't see any review, just you commenting on some nitpick...

Reply:
> I don't see any review, just you commenting on some nitpick...

Sorry if I didn't write it clearly: I would leave an approval if the nitpick is addressed.

Reply:
Sorry, but the title of the added commit "Fix a type in a comment" has a typo.

`type` -> `typo`